dime-coding-standards/checklists/tex/code-review-package-checklist

```diff
@@ -0,0 +1,133 @@
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Code Review Package Checklist}} \\
+				\Large\textbf{{v2.0}}}
+		}
+		
+		\section*{Required Content and Practices}
+		
+		The code review package must be shared as a .zip file, GitHub link or shared folder containing the items listed below.
+		All file paths in the code \textbf{must} use only forward slashes, i.e. \texttt{C:/users/...}
+		
+		\subsection{1. \, Package components}
+		
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{All scripts to be reviewed.}
+			\item[] \dimeCheckBox{All data necessary to run the code to be reviewed - in deidentified form.}
+			\item[] \dimeCheckBox{A README. file that contains the following information:}
+                        \begin{itemize}
+                            \item[] \dimeCheckBox{Purpose of each folder}
+                            \item[] \dimeCheckBox{Purpose of each script (if more than one script is used)}
+                            \item[] \dimeCheckBox{Is all confidential data de-identified? (if provided in the review)} 
+                             \item[] \dimeCheckBox{Outputs produced (if part of the review)}
+                             \item[] \dimeCheckBox{Which script creates which output}
+                             
+                            
+                        \end{itemize}
+			\item[] \dimeTextField{master}{11cm}{README. file path and name:}
+			\item[] \dimeCheckBox{A \textbf{master script} that runs all the necessary code and allows the reviewer to adjust file paths only once.}
+		\end{itemize}
+		
+		\vspace{.6em}
+		
+		\subsection{2. \, Master script requirements}	
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{ Master script clearly indicates which line needs to be changed to run in a different computer}
+			\item[] \dimeCheckBox{ The master script runs from start to end if the reviewer changes only directory path(s) in one location of the master script}
+			\item[] \dimeCheckBox{ The master script installs any commands required (e.g. from SSC in Stata, CRAN in R, etc.)}
+			\item[] \dimeCheckBox{ In Stata, the master do file sets critical configurations such as version, matsize, and varabbrev (either sets directly or through a wrapper command such as ieboilstart from ietoolkit}
+			\item[] \dimeCheckBox{ In R, all necessary packages to run the code are loaded in the master script}
+			\item[] \dimeTextField{time}{2.5cm}{\textbf{Approximate time} it takes to run all code from the the master script: }
+		\end{itemize}
+		
+		\subsection{2. \, Data}
+		
+		The package includes:
```

Comment:
for peer review, i think the first question is 'are you able to share the data for the purposes of peer review?' if yes, then there is an additional checkbox to confirm that it is de-identified and that all necessary datasets are shared. if not, then the reviewer knows they are looking at the code only, not trying to run the files. 

I suggest erasing the point "Any exception where sensitive data is strictly needed to conduct the review will require additional ethics and security measures, and will be handled on a case-by-case basis." -- there is no need to go through this for peer review, they can just share the code for feedback. additional measures are only worth it for required reproducibility checks.